src/main/flight/mixer.c

```diff
@@ -607,9 +621,25 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
     // Find min and max throttle based on conditions. Throttle has to be known before mixing
     calculateThrottleAndCurrentMotorEndpoints(currentTimeUs);
 
+    static bool turtleModeStarted = false;
     if (isFlipOverAfterCrashActive()) {
-        applyFlipOverAfterCrashModeToMotors();
+        static float tiltAngleAtStart = 1.0f;
+        float flipAngleModifier = 1.0f;
+#ifdef USE_ACC
+        const float tiltAngle = getCosTiltAngle(); // -1 if inverted, 0 when 90 degrees, 1 when flat and upright
+        if (!turtleModeStarted) {
+            tiltAngleAtStart = tiltAngle;
+        }
+        turtleModeStarted = true;
```

Comment:
This only works once. On subsequent turtle mode activations this will not be triggered anymore.